src/generics.rst

```diff
@@ -173,9 +173,39 @@ A :t:`constant parameter` shall be used in the following contexts:
   As a :t:`value` of any :t:`expression` within a :t:`function body`.
 
 :dp:`fls_hidfwkwr2r73`
-A :t:`type parameter` has an implicit :std:`core::marker::Sized` bound, unless a
+A :t:`type parameter` has an implicit :std:`core::marker::Sized` :t:`bound`, unless a
 ``?core::marker::Sized`` bound is present.
 
+:dp:`fls_yaYedRNaxhTe`
+A :t:`type parameter` declared on an :t:`abstract data type` has implicit
```

Comment:
How do you like

A type parameter of an abstract data type ...

I find "declared on" a bit weird.